docs/frontend/backend-layouts.md

```diff
@@ -6,21 +6,25 @@ In the backednd layout you can define multiple columns where you want to put you
 
 If you would like to create multiple columns backend layout just define your layout and create some Vue component. Your API for selected page should returns 
 
-`...page.appearance`:
+`(partial) page response`:
 ``` json{4,5}
 {
   ...
   "content": {
     "colPos0": [...Array with content elements],
     "colPos1": [...Array with content elements]
-  }
+  },
+  "appearance": {
+    "layout": "layout-0",
+    "backendLayout": "2Columns"
+  },
   ...
 }
 ```
 
-If the response is ok, we can create component file. 
+If the response is ok - we see there are 2 object in content and custom backendLayout, then we can create layout file. To render content from selected `colPos` you can use `T3Renderer` .
```

Comment:
We can see 2 objects - `content` and `appearance` with `backendLayout` property.

If the response is ok, we will receive 2 objects - `content` and `appearance`. The name of the backend layout is stored in the `appearance` object as `backendLayout` property, so we should create a layout file named after data provided by API.
To render content from selected `colPos` you can use `T3Renderer`.